### DISCUSSION TITLE
Questions / Observations - almost 1 Year with Solar.

### DISCUSSION THREAD

**Discussion by wreiske** [jasonacox/Powerwall-Dashboard#359] [Q&A]
Hey everyone!

I've been dealing with Tesla customer support for well over a year now. To make a very long story short, they messed up on the planning on my Solar system, the install of the system, and now they just ignore all my emails and support requests. I've called them out on several mistakes they have made along the way and they have not done anything to make it right. I'll eventually share all of my emails back and forth with them, the phone call logs, etc., but that's for another day...

I'm looking back at my solar production for the last year. I'm located in Arizona and have an 18.4 kW system with 2 Powerwall+. I have a few questions that you all may be able to answer...

<img width="1663" alt="Screenshot 2023-09-24 at 4 46 34 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/dc7189e4-b392-4090-9d34-77afeb2be881">

1. I've noticed that I get more solar power when there are some clouds in the sky rather than direct sunlight with no cloud coverage. I believe the peak 15.4 kW I experienced was during a cloudy day (See screenshot below). Does a single burst of 15.4 kW on an 18.4 kW system seem accurate for Arizona?

<img width="1107" alt="Screenshot 2023-09-24 at 4 57 39 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/7af1ca63-a22c-40b2-adac-82d0f30adcf5">

2. There are strings with almost now power. Why?

<img width="1641" alt="Screenshot 2023-09-24 at 4 45 38 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/90c1cdb9-95bd-4f8a-a27e-ec267428f084">
<img width="393" alt="Screenshot 2023-09-24 at 5 02 03 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/df625089-6df6-4f14-9ecf-dad9870b0cb2">
<img width="1638" alt="Screenshot 2023-09-24 at 5 03 00 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/a563cdc9-15f3-44d3-a155-5e35f500bc9d">

<img width="1640" alt="Screenshot 2023-09-24 at 5 03 19 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/b322815d-1ef3-4a72-a960-1343c9f2e69b">

B1 for comparison
<img width="1643" alt="Screenshot 2023-09-24 at 5 03 44 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/9dbb7bf4-e88e-4364-a27e-57d2d182417e">

I may be completely wrong in assuming I should be getting more power out of an 18.4kW system in the middle of summer in AZ. Here's the summer equinox for 2023:

<img width="1104" alt="Screenshot 2023-09-24 at 5 12 20 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/245421a9-3db7-4378-8c58-0c80fcd35f6b">

They do say they are rated 80% of their output? When I purchased this system, it was originally supposed to be over 19kW but they made a mistake when planning the placement on my roof and had to put a smaller system on (they had marked a part of my roof as having something in the way that doesn't actually exist on my roof...).. When I pointed that out to them they stopped responding to my emails...

Here's my last 24 hours of alerts, just in case this is useful for anyone.
<img width="1635" alt="Screenshot 2023-09-24 at 5 15 00 PM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/6295044/4b178423-6a1a-47c5-8955-4c8490e47449">

Looking forward to seeing if anyone has some answers. Tesla has been no help at all.

**Comment by jasonacox** [comment]
Hi @wreiske !

> I've noticed that I get more solar power when there are some clouds in the sky rather than direct sunlight with no cloud coverage. 

That is called "cloud lensing" - It basically directs sunlight that would normally not focus on your roof to temporarily bend toward your roof. You will see these as temporary kW spikes.

> There are strings with almost now power. Why?

Are these actually connect strings? I have one that floats in the small voltage ranges but it is not really connected.

> They do say they are rated 80% of their output? 

It's complicated. You are right. The angle of the panels and surrounding obstructions will impact production. The way to figure the kW of your system is to count the number of panels you have and multiple it by the panel rating. I have 20 x 425W panels which is a 8.5kW system.  My inverter is rated at 7.6kW so it will clip before 8.5kW.  I've seen 7.2kW but normally it stays in the 6kW range. You will want to find out the limits on each of your strings based on the inverter they connect to.

Your system looks healthy.  Based on the panel angles on your roof, it could be performing as expected.  What is the peak you see?  13kW?  Keep in mind what really matters is kWh as the different strings/angles will be optimizing (peaking) for different times of the day. I would look at the kWh that Tesla predicted for your installation to see if you are on track with the design and not worry as much about the peak KW.

**Comment by MattMake** [comment]
I hear ya, I'm kind of in the same boat.  I have 20.4kW solar and (2) 7.8kW inverters.  The max I see (on partly cloudy days) is 17.2kW for short spurts, on sunny days it is more around 15.2kW steady.  The majority of that is due to the inverter limitations.  It also has to do with the way Tesla connected my panels.  
I have 11 panels facing east, 11 facing west, and 29 facing south.  The designer had split the 29 panels (after a small 3 panel array to get an even number) + the east panels, and the other half with the west facing panels.
The installers, however, connected it in the way that worked best for them installing.  They put 26 of the south facing panels in a parallel string on one inverter, and 3 panels south + 11 east + 11 west on the second inverter.
That means I have one inverter running in overdrive, and the other is twiddling it's thumbs.  
However, my system was estimated to generate 22,951kWh per year, and I've completed my first year about a month ago having generated 28,610kWh.  And I'm presently at 123% energy offset.
Do I wish they had configured the panel strings better?  Absolutely.  But, I know having them come out and re-arrange is going to take an act of Musk.  It is running above what they projected, so I can't complain too much.
